docker/cassandra.yaml

```diff
@@ -1,916 +1,92 @@
-# Cassandra storage config YAML
-
-# NOTE:
-#   See http://wiki.apache.org/cassandra/StorageConfiguration for
-#   full explanations of configuration directives
-# /NOTE
-
```

Comment:
Removed a ton of cruft, this shouldn't be apart of the config on the docker container...